bindings/python/iota_sdk/types/burn.py

```diff
@@ -58,6 +58,15 @@ def add_native_token(self, native_token: NativeToken) -> Burn:
         return self
 
     def as_dict(self) -> Dict[str, Any]:
+        """
```

Comment:
Was this forgotten to change to simply `Converts this object to a dict.` or is that a legit specific one?

Reply:
Missed it